web/js/signup_comps/Auth.js

```diff
@@ -68,16 +83,100 @@ var SocialAuth = React.createClass({
     }
 });
 
+var wasClicked = {
+  "Google": null,
+  "Facebook": null,
+  "Twitter": null,
+  "Tumblr": null,
+  "Reddit": null,
+  "Instagram": null,
+  "Spotify": null
+};
+var sharePoints = 0;
+var clickBtns = 0;
```

Comment:
fixed

On Mon, Feb 15, 2016 at 11:22 PM, Micha Gorelick <notifications@github.com>
wrote:

> In web/js/signup_comps/Auth.js
> <https://github.com/QuantifiedSelfless/quantifiedSelf/pull/47#discussion_r52972362>
> :
>
> > @@ -68,16 +83,100 @@ var SocialAuth = React.createClass({
> >      }
> >  });
> >
> > +var wasClicked = {
> > +  "Google": null,
> > +  "Facebook": null,
> > +  "Twitter": null,
> > +  "Tumblr": null,
> > +  "Reddit": null,
> > +  "Instagram": null,
> > +  "Spotify": null
> > +};
> > +var sharePoints = 0;
> > +var clickBtns = 0;
>
> clickBtns and sharePoints don't seem to be used (nor should they be since
> wasClicked has all the info we need)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/QuantifiedSelfless/quantifiedSelf/pull/47/files#r52972362>
> .
>